docker/local/deployment/nginx-based/docker-compose.yml

```diff
@@ -0,0 +1,245 @@
+version: '3.9'
+services:
+  redis:
+    image: 'redis:alpine'
+    container_name: redis
+    restart: unless-stopped
+    logging:
+      driver: 'none'
+
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    logging:
+      driver: 'json-file'
+      options:
+        max-size: '50m'
+        max-file: '5'
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
+    volumes:
+      - mongodb:/data/db
+    ports:
+      - 27017:27017
+    restart: unless-stopped
+  
+  api:
+    image: 'ghcr.io/novuhq/novu/api:latest'
+    depends_on:
+      - mongodb
+      - redis
+      - nginx-proxy
+      - nginx-proxy-acme
```

Comment:
`nginx-proxy-acme` depends on `nginx-proxy`, so we can just have `nginx-proxy-acme` here and in some other places as well